sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -1033,14 +1034,14 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       // Only running workflow instances can be deleted.
       if (application.assessmentWorkflowId && !application.hasAssessmentData) {
```

Comment:
Not a part of the PR. But I have a question here with camunda 8 implementation. When an application is completed the workflow will be set to end state here as we do not have the human task like in camunda 7. And so is expected to happen for reassessment cycle that workflow will be set to end state. Hence while cancelling applications which are not in completed status will be more accurate here right? `application.assessmentWorkflowId && application.applicationStatus === ApplicationStatus.cancelled`

Reply:
The first point, to be clear, the human task was just a workaround and it would never be there for PROD no matter Camunda 7 or 8.
The `if` can be improved but right now it is still valid to check if the workflow is "running" or not, `application.hasAssessmentData` is just a way to have a better guess if it is running or now. 
If needed we can have a status in the assessment to mark it as `Completed` from a workflow perspective.